projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, ValidationErrors, ValidatorFn, Validators } from '@angular/forms';
+import { DataLayerManagerService } from 'ngx-charts-on-fhir';
+
+const BloodPressureRangeValidator: ValidatorFn = (control: AbstractControl): ValidationErrors | null => {
+  const systolic = control.get('systolic');
+  const diastolic = control.get('diastolic');
+
+  return systolic && diastolic && systolic.value > diastolic.value ? null : { bloodPressure: true };
+};
+
+@Component({
+  selector: 'report-bp',
+  templateUrl: './report-bp.component.html',
+  styleUrls: ['./report-bp.component.css'],
+})
+export class ReportbpComponent implements OnInit {
```

Comment:
To follow naming convention for classes/types I think BP should be upper case: `ReportBPComponent`